sass/_grid.scss

```diff
@@ -0,0 +1,58 @@
+/* grid */
+
+@mixin row($inside: false) {
+  @if($inside == true) {
+    margin-left: -$gutter/2;
+    margin-right: -$gutter/2;
+  }
+  @else {
+    width: 100%;
+    max-width: $wrapper;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  &:before, &:after {
```

Comment:
Feel free to create a clearfix mixin yourself and reuse it later within the project.